src/main/kotlin/com/hubelias/parkingmeter/application/MoneyDto.kt

```diff
@@ -0,0 +1,9 @@
+package com.hubelias.parkingmeter.application
+
+
+data class MoneyDto(val amount: Double, val currency: Currency) {
```

Comment:
dlaczego tutaj double?

Reply:
MoneyDto jest zwracane z serwisu aplikacji, nigdy nie jest wykorzystywane do obliczeń (te są wykonywane na BigDecimalach przez joda-money). Uznałem, że skoro konwersja na DTO następuje już po obliczeniach, to nie ma wielkiej różnicy, czy tu będzie double, czy big decimal, a z doublem zapis jest bardziej zwięzły i lepiej wygląda w testach.